site/_data/config/bk_server.yaml

```diff
@@ -38,6 +47,26 @@ groups:
   - param: lostBookieRecoveryDelay
     description: How long to wait, in seconds, before starting autorecovery of a lost bookie.
     default: 0
+  - param: allowEphemeralPorts
+    description: Whether the bookie allowed to use an ephemeral port (port 0) as its server port. By default, an ephemeral port is not allowed. Using an ephemeral port as the service port usually indicates a configuration error. However, in unit tests, using ephemeral port will address port conflicts problem and allow running tests in parallel.
+    default: 'false'
+  - param: enableLocalTransport
+    description: Whether conf the bookie to listen for BookKeeper clients executed on the local JVM.
```

Comment:
The meaning of this sentence isn't quite clear. There seems to be maybe an extra noun?